ansible/roles/clone_repo/tasks/main.yml

```diff
@@ -8,7 +8,7 @@
   git:
     repo: https://github.com/avmikholap/dos14-Miholap_Aleksey_git-flow.git
     dest: /home/authz/git
-    version: feature-hw-25
+    version: feature-hw-26
```

Comment:
Lets add ansible variable here with which we can tell what git branch to pull

Reply:
added variable ansible for the branch